po/cs.po

```diff
@@ -2892,9 +2892,9 @@ msgid ""
 "  \\\\\\\\\\\\\\__o\n"
 "__\\\\\\\\\\\\\\'/_"
 msgstr ""
-"   \\\\\\\\\\\n"
-"  \\\\\\\\\\\\\\__o\n"
-"__\\\\\\\\\\\\\\'/_"
+"  (o)(o)--."
+"   \oo/ (  )"
+"   m\/m--m'`--------."
```

Comment:
```suggestion
"   m\\/m--m'`--------."
```
Inside a Cstring a literal BackSlash (\\) needs to be escaped `\\`.
(No NewLine char on the last line.)"